tools/check_spelling.sh

```diff
@@ -27,21 +33,25 @@ SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
 ROOTDIR=$SCRIPTPATH/..
 cd "$ROOTDIR"
 
+BIN_FILENAME="misspell_${VERSION}_${OS}_64bit.tar.gz"
 # Install tools we need
-if [[ ! -e "/tmp/misspell"  ]];then
-  wget -qO- https://github.com/client9/misspell/releases/download/v${VERSION}/misspell_${VERSION}_${OS}_64bit.tar.gz|tar xvz -C /tmp &> /dev/null
+if [[ ! -e "/tmp/misspell" ]]; then
+  if ! wget https://github.com/client9/misspell/releases/download/v${VERSION}/${BIN_FILENAME} -O /tmp/${BIN_FILENAME} --no-verbose -t 3  -o /tmp/wget.log; then
```

Comment:
OK this is fine. I think would also be fine to just do a simple wget with no flags and let the output show. It only will be run occasionally, and when it does need to access the network it would be good to know that is what it is doing.

Also please use `--tries=3` to optimize for readability.